components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -270,7 +270,7 @@
 <config_flux_attenuation_coefficient_runoff>10.0</config_flux_attenuation_coefficient_runoff>
 
 <!-- coupling -->
-<config_ssh_grad_relax_timescale>86400.0</config_ssh_grad_relax_timescale>
+<config_ssh_grad_relax_timescale>0.0</config_ssh_grad_relax_timescale>
```

Comment:
If we are removing SSH filtering, it makes more sense to remove these computations, rather than set the coefficient to zero. 
```
components/mpas-ocean/driver/ocn_comp_mct.F
1088          if (config_ssh_grad_relax_timescale < real(ocn_cpl_dt,RKIND)) then
1089             timeFilterFactor = 1.0_RKIND
1090          else
1091             timeFilterFactor = real(ocn_cpl_dt,RKIND) / config_ssh_grad_relax_timescale
1092          endif
1093          filteredSSHGradientZonal = filteredSSHGradientZonal * (1.0_RKIND - timeFilterFactor) + &
1094               avgSSHGradient(index_avgZonalSSHGradient, :) * timeFilterFactor
1095          filteredSSHGradientMeridional = filteredSSHGradientMeridional * (1.0_RKIND - timeFilterFactor) + &
1096               avgSSHGradient(index_avgMeridionalSSHGradient, :) * timeFilterFactor
```
The best way to do this is to remove the `filteredSSHGradient*` variables completely. But if we would like to have the option to filter in the future, we could add a logical, and if it is false, then
```
filteredSSHGradientZonal =  avgSSHGradient(index_avgZonalSSHGradient, :)
filteredSSHGradientMeridional =  avgSSHGradient(index_avgMeridionalSSHGradient, :)
```

I think setting the timescale to zero is only appropriate as a temporary solution to remove the filtering.

Reply:
For reducing unneeded flops, this is a suitable solution
```
components/mpas-ocean/driver/ocn_comp_mct.F
1088          if (config_ssh_grad_relax_timescale < real(ocn_cpl_dt,RKIND)) then
                filteredSSHGradientZonal =  avgSSHGradient(index_avgZonalSSHGradient, :)
                filteredSSHGradientMeridional =  avgSSHGradient(index_avgMeridionalSSHGradient, :)
1090          else
1091            timeFilterFactor = real(ocn_cpl_dt,RKIND) / config_ssh_grad_relax_timescale
1092          
1093            filteredSSHGradientZonal = filteredSSHGradientZonal * (1.0_RKIND - timeFilterFactor) + &
1094               avgSSHGradient(index_avgZonalSSHGradient, :) * timeFilterFactor
1095            filteredSSHGradientMeridional = filteredSSHGradientMeridional * (1.0_RKIND - timeFilterFactor) + &
1096               avgSSHGradient(index_avgMeridionalSSHGradient, :) * timeFilterFactor
              endif
```

Reply:
Done.